shudder/queue.py

```diff
@@ -19,37 +19,38 @@
 """
 import json
 
-from boto.utils import get_instance_metadata
-import boto.sns as sns
-import boto.sqs as sqs
+import boto3
```

Comment:
There is no ''get_instance_metadata'' in boto3. Added a convenience function to metadata.py